src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspace.cs

```diff
@@ -19,6 +19,7 @@ public abstract class VisualStudioWorkspace : Workspace
     {
         private BackgroundCompiler _backgroundCompiler;
         private readonly BackgroundParser _backgroundParser;
+        internal static bool AllowUseOfHierarchy;
```

Comment:
This is just for testing, because the appropriate events weren't available in the drop I was using - this will go away before merging.